monai/bundle/scripts.py

```diff
@@ -608,16 +608,19 @@ def run(
     .. code-block:: bash
 
         # Execute this module as a CLI entry:
-        python -m monai.bundle run training --meta_file <meta path> --config_file <config path>
+        python -m monai.bundle run --meta_file <meta path> --config_file <config path>
+
+        # Execute with specified `run_id=runtest`, `init_id=inittest`, `final_id=finaltest`:
+        python -m monai.bundle run --run_id runtest --init_id inittest --final_id finaltest ...
 
         # Override config values at runtime by specifying the component id and its new value:
-        python -m monai.bundle run training --net#input_chns 1 ...
```

Comment:
If you remove all examples where you're providing `run_id` as a value on the command line it's not clear to the user that this is an option. Old examples of command lines or those bundles not using specified values won't work so I'd suggest leaving one example here of providing `run_id` explicitly and state what this means.